packages/core/strapi/lib/commands/content-types/generate-types.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const tsUtils = require('@strapi/typescript-utils');
+
+const strapi = require('../../index');
+
+module.exports = async function({ outDir, file, debug }) {
+  const appContext = await strapi.compile();
```

Comment:
Should we say anything if you don't have a typescript project ? Like suggestion moving to typescript to really benefit from the types or sth ?

Reply:
I don't think so, in the future if we provide typings for our core APIs (entity service, etc..), they'll still read the schema.d.ts file and provide accurate typings even if the project is in JavaScript.